docs/tools/terraform/v1v2.rst

```diff
@@ -0,0 +1,216 @@
+Upgrading the Aiven Terraform Provider from v1 to v2
+====================================================
+
+Version 2 of the Aiven Terraform Provider was released in `October of
+2020 <https://aiven.io/blog/aiven-terraform-provider-v2-release>`__. You
```

Comment:
I think it would be better to say "We recommend you upgrade to v2 if you haven't already, and this guide will walk you through the process". Don't reference other possible approaches unless we have docs or something else to point to.